Sources/AWSLambdaRuntimeCore/LambdaHandler.swift

```diff
@@ -157,7 +157,7 @@ extension EventLoopLambdaHandler where Output == Void {
 /// - note: This is a low level protocol designed to power the higher level ``EventLoopLambdaHandler`` and
 ///         ``LambdaHandler`` based APIs.
 ///         Most users are not expected to use this protocol.
-public protocol ByteBufferLambdaHandler {
```

Comment:
IMO the `ByteBufferLambdaHandler` does not need to be sendable. The runtime will always call it in a serial manner. We should wrap the protocol in a `UncheckedSendableBox`.